### ISSUE TITLE
Specify which module notifications refer to when a CVE is found

### ISSUE THREAD

**Issue by agatti** [tonybaloney/pycharm-security#259]
# Feature Request

When checking packages, include in which module issues are found when reporting them.

If you happen to have a project that is composed of several separate modules attached to a root project, when issues are found in a module dependency then it becomes quite tedious to open every requirements.txt/Pipfile/pyproject.toml/etc. in the project to find out where those are.

For example, here's the event log on a project of mine (popups don't say which project they refer to either):

<img width="1581" alt="Screenshot 2021-08-09 at 12 15 04" src="https://user-images.githubusercontent.com/235740/128691647-2573d355-49b8-46b3-8346-f5f85e102afe.png">

This is for PyCharm only I guess - although if you have nested projects in a GitHub Action context this may apply there too?

**Comment by tonybaloney** [comment]
This is a great idea, but I can't see how to get this information out of the PyCharm API. It only gives me the location of where the package is installed (which I don't think would help)

**Comment by agatti** [comment]
Right, I've had a go myself as well and didn't really get anywhere in a way that would work on all supported package managers in the base installation (and Poetry via its third party plugin).  Should this be filed on PyCharm's YouTrack instead?

**Comment by tonybaloney** [comment]
Discussed in #385  (the API has changed)

**Comment by tonybaloney** [comment]
Released in 1.28.0
